crowbar_framework/app/controllers/network_controller.rb

```diff
@@ -125,22 +125,28 @@ def enable_interface
 
   add_help(:disable_interface, [:id, :network, :name], [:post])
   def disable_interface
-    code, message = @service_object.disable_interface(
-      params[:id],
-      params[:network],
-      params[:name]
-    )
-
     respond_to do |format|
-      case code
-      when 200
-        format.json { head :ok }
-      else
-        format.json do
-          render json: { error: message }, status: code
-        end
+      format.json do
+        render json: { error: "Disabling is unsupported" }, status: 500
       end
     end
+
+    # code, message = @service_object.disable_interface(
+    #   params[:id],
+    #   params[:network],
+    #   params[:name]
+    # )
+
+    # respond_to do |format|
+    #   case code
+    #   when 200
+    #     format.json { head :ok }
+    #   else
+    #     format.json do
+    #       render json: { error: message }, status: code
+    #     end
+    #   end
+    # end
```

Comment:
Do we want commented code when we are using git? If this is WIP code I guess that is more correct to uncomment and stash that having it here